app/services/ccms/requestors/base_requestor.rb

```diff
@@ -0,0 +1,101 @@
+module CCMS
+  module Requestors
+    class BaseRequestor
+      NAMESPACES = {
+        "xmlns:bill" => "http://legalservices.gov.uk/CCMS/Finance/Payables/1.0/BillingBIO",
+        "xmlns:casebim" => "http://legalservices.gov.uk/CCMS/CaseManagement/Case/1.0/CaseBIM",
+        "xmlns:casebio" => "http://legalservices.gov.uk/CCMS/CaseManagement/Case/1.0/CaseBIO",
+        "xmlns:clientbio" => "http://legalservices.gov.uk/CCMS/ClientManagement/Client/1.0/ClientBIO",
+        "xmlns:clientbim" => "http://legalservices.gov.uk/CCMS/ClientManagement/Client/1.0/ClientBIM",
```

Comment:
not sure how far to sweat this but technically these "references" are not needed as the single subclass only uses `common` and `refdatabim` to contruct it payload. The casebim/o and clientbim/o are for case adding and client/applicant adding respectively, which are unlikely to ever be used by LFA

Reply:
👍 have removed unmatched namespace references